modules/nodes-nodes-garbage-collection-configuring.adoc

```diff
@@ -117,8 +117,8 @@ spec:
 For `evictionHard` you must specify all of these parameters.  If you do not specify all parameters, only the specified parameters are applied and the garbage collection will not function properly.
 <7> For container garbage collection: The duration to wait before transitioning out of an eviction pressure condition. Setting the `evictionPressureTransitionPeriod` parameter to `0` configures the default value of 5 minutes.
 <8> For image garbage collection: The minimum age for an unused image before the image is removed by garbage collection.
-<9> For image garbage collection: The percent of disk usage (expressed as an integer) that triggers image garbage collection.
-<10> For image garbage collection: The percent of disk usage (expressed as an integer) that image garbage collection attempts to free.
+<9> For image garbage collection: The percent of disk usage (expressed as an integer) that triggers image garbage collection. This value must be greater than the `imageGCLowThresholdPercent` value.
+<10> For image garbage collection: The percent of disk usage (expressed as an integer) that image garbage collection attempts to free. This value must be less than the `imageGCHighThresholdPercent` value.
```

Comment:
I think it's more correct to modify as: 
<10> For image garbage collection: The percent of disk usage (expressed as an integer), to which image garbage collection attempts to free

Reply:
others looks good to me